package-lock.json

```diff
@@ -2062,21 +2047,21 @@
       "dev": true
     },
     "eslint": {
-      "version": "4.18.0",
-      "resolved": "https://registry.npmjs.org/eslint/-/eslint-4.18.0.tgz",
-      "integrity": "sha512-Ep2lUbztzXLg0gNUl48I1xvbQFy1QuWyh1C9PSympmln33jwOr8B3QfuEcXpPPE4uSwEzDaWhUxBN0sNQkzrBg==",
+      "version": "4.19.1",
+      "resolved": "https://artifactory-dev.exacttarget.com:443/artifactory/api/npm/sfmc-npm/eslint/-/eslint-4.19.1.tgz?dl=https://registry.npmjs.org/eslint/-/eslint-4.19.1.tgz",
```

Comment:
@infolock Are there any concerns with using the Artifactory URL in here?

Reply:
ah - good call.